File: ruby/retrieve-linked-content/GetPageWithSubpages.rb
# DocSection: linked_content_get_page_with_subpages
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '8d20758c-d74c-4f59-ae04-ee928c0816b7'
delivery_client.item('insurance_listing')
               .depth(1)
               .execute do |response|
                 item = response.item
                 puts response.json
               end
# EndDocSection

File: js/retrieve-linked-content/GetPageWithSubpages.js
// DocSection: linked_content_get_page_with_subpages
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  projectId: '8d20758c-d74c-4f59-ae04-ee928c0816b7'
});

const response = await deliveryClient.item('insurance_listing')
  .depthParameter(1)
  .toPromise();
// EndDocSection